server/src/main/java/com/broadcom/lsp/cobol/domain/event/model/RetrieveRangeEvent.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2020 Broadcom.
```

Comment:
It's 2021 now

Reply:
yes, but I can't set 2021, because it requires 2020

Reply:
Sorry for that, it must be fixed by #787

Reply:
As agreed with Scott, PO we keep 2020 there, no change to 2021